framework/lib/utils/get-consistent-random-color-from-string/index.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-bitwise */
+import { keys, omit } from 'ramda'
+import { palette } from '@northlight/tokens'
+
+const availableColorSchemes = keys(omit([ 'mono' ], palette))
+
+const djb2Hash = (str: string, colors: string[]) => {
+  let hash = 5381
+
+  for (let i = 0; i < str.length; i += 1) {
+    const char = str.charCodeAt(i)
+    hash = (hash << 5) + hash + char
+
+    hash &= hash
+  }
+
+  return (Math.abs(hash) % colors.length) + 1
```

Comment:
Will this ever generate the 0th index of the colors array?

Reply:
Good observation, I removed the +1